templates/exporter/exporter-dpl.yaml

```diff
@@ -64,6 +64,13 @@ spec:
               name: {{ .Values.database.external.existingSecret }}
               key: password
         {{- end }}
+        {{- if .Values.existingSecretAdminPassword }}
```

Comment:
Same as above. Keep it simple by adding a line to the existing secret.

Reply:
same as previous comment